server/service/grpc/error.go

```diff
@@ -11,4 +11,5 @@ var (
 	ErrBindHeartbeatStream   = coderr.NewCodeError(coderr.Internal, "bind heartbeat sender")
 	ErrUnbindHeartbeatStream = coderr.NewCodeError(coderr.Internal, "unbind heartbeat sender")
 	ErrForward               = coderr.NewCodeError(coderr.Internal, "grpc forward")
+	ErrOperation             = coderr.NewCodeError(coderr.Internal, "grpc operation")
```

Comment:
```suggestion
	ErrGrpcFlowLimite             = coderr.NewCodeError(coderr.Internal, "grpc request volume reaches the threshold")
```
Make limiter err info more clear.